### DISCUSSION TITLE
BIlling Github Teams for organization through Azure

### DISCUSSION THREAD

**Discussion by pchambongard** [community/community#84962] [Enterprise]
Hello!

I'm trying to upgrade my organization to Github Teams, and billing through Azure Subscription.

I already did azure subscription for Github Copilot for some member of our organization, but at checkout when upgrading to teams or Enterprise, Github asks me for a credit card or a PayPal Account, and there is no option to pay by Azure, 

Does someone know how to do that?

Thanks a lot,

Pierre

**Comment by akashdip2001** [comment]
GitHub typically handles billing and subscriptions independently of third-party services like Azure. While GitHub may integrate with Azure DevOps for certain functionalities, the billing for GitHub plans, including GitHub Teams or GitHub Enterprise, is typically managed directly on the GitHub platform.

If you're looking to upgrade your organization to GitHub Teams and want to use your Azure subscription for billing, here are some steps you can follow:

1. **Contact GitHub Sales:**
   - Reach out to GitHub Sales or support to inquire about billing options and the possibility of using your Azure subscription. They can provide the most accurate and up-to-date information.

2. **GitHub Enterprise with Azure Active Directory (AAD):**
   - If you are interested in GitHub Enterprise and integration with Azure, GitHub Enterprise Server supports authentication using Azure Active Directory. However, the billing may still be separate.
   - You might consider GitHub Enterprise Server with Azure Active Directory authentication for user management while maintaining the GitHub subscription separately.

3. **Review GitHub Documentation:**
   - Check GitHub's official documentation or Azure Marketplace documentation for any recent updates regarding billing integration.
   - GitHub's features and billing options can evolve, so staying updated with the latest documentation is crucial.

4. **Azure DevOps Services:**
   - If you are looking for tighter integration between GitHub and Azure, consider using Azure DevOps Services. GitHub and Azure DevOps have some integrations, and Azure DevOps Services has its own billing model.

Remember to consult with GitHub Sales or support directly for the most accurate and personalized assistance based on your organization's needs. They can guide you through the available options and help you make informed decisions regarding billing and subscriptions.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by pchambongard** [reply]
Thanks for your answer, 

I understand that Github Teams or Enterprise might not be considered as metered Services

But I still have the same problem for example, when I try to set spending limit for Github Packages, I have a Azure token linked to my GitHub organization, it says All metered services are being billed through Azure, But yet I can't set a spending limit on GitHub packages without putting a card or a PayPal account, 
Maybe I missed some buttons or hidden checkboxes? 

Because in the documentation it clearly says that Github Packages can be Billed via Azure

"If you are an organization owner, you can connect an Azure Subscription ID to your organization account to enable and pay for GitHub Packages usage beyond the amounts including with your account. For more information, see "[Connecting an Azure subscription](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription)."

Thanks again,

**Reply by teemuhonkanen** [reply]
did you get any answer on this?

**Reply by pchambongard** [reply]
Right now in my organization only Github Copilot is billed via Azure

**Comment by nick3** [comment]
I also need to use an Azure subscription to pay for GitHub Team.

**Comment by gcavaglierivg** [comment]
I haven’t been able to find any official Microsoft/GitHub source that clearly states this, but based on the tests I’ve done, I can confirm that GitHub Teams cannot be billed through an Azure subscription. It’s not really designed for enterprise scenarios, so the only payment methods available are credit card or PayPal.

On the other hand, GitHub Enterprise can be billed via an Azure subscription, since it’s intended for organizations and can be managed at the organizaton level.

For context, I ran my tests with an Azure CSP subscription, but I don’t believe the subscription type (CSP vs. EA vs. PAYG) makes much difference here.
